tests/integration/test_merge_tree_optimize_old_parts/test.py

```diff
@@ -13,7 +13,7 @@
 
 
 @pytest.fixture(scope="module")
-def start_cluster():
+def started_cluster():
```

Comment:
BTW it looks like this test can be replaced with a simple stateless test (`.sql`) ?

Reply:
not sure how to wait for merge apart from doing sleep, any ideas?

Reply:
If you need you can sleep with a query - `select sleep(3)` or if you need more then 3 second `select sleepEachRow(1) from numbers(10)`